src/moi_oracle.jl

```diff
@@ -53,6 +53,289 @@ function compute_extreme_point(
+        var_constraint_list[func] = idx
+    end     
+end
+
+function compute_inface_extreme_point(lmo::MathOptLMO{OT}, direction, x; kwargs...) where {OT}
+    var_constraint_list = Dict([])
+    lmo2 = copy(lmo)
+    MOI.empty!(lmo2.o)
+    MOI.set(lmo2.o, MOI.Silent(), true)
+    variables = MOI.get(lmo.o, MOI.ListOfVariableIndices())
+    MOI.add_variables(lmo2.o, length(variables)) 
+    terms = [MOI.ScalarAffineTerm(d, v) for (d, v) in zip(direction, variables)]
+    obj = MOI.ScalarAffineFunction(terms, zero(Float64))
+    MOI.set(lmo2.o, MOI.ObjectiveFunction{typeof(obj)}(), obj)
+    for (F, S) in MOI.get(lmo.o, MOI.ListOfConstraintTypesPresent())
+        valvar(f) = x[f.value]
+        const_list = MOI.get(lmo.o, MOI.ListOfConstraintIndices{F,S}())
+        for c_idx in const_list
+            func = MOI.get(lmo.o, MOI.ConstraintFunction(), c_idx)
+            val = MOIU.eval_variables(valvar, func)
+            set = MOI.get(lmo.o, MOI.ConstraintSet(), c_idx)
+            set_constraint(lmo2.o, S, func, val, set, var_constraint_list)  
+        end
+    end
+    MOI.set(lmo2.o, MOI.ObjectiveSense(), MOI.MIN_SENSE)
+    MOI.optimize!(lmo2.o)
+    a = MOI.get(lmo2.o, MOI.VariablePrimal(), variables)
+    MOI.empty!(lmo2.o)
+    return a
+end
+
+# Second version of compute_inface_extreme_point.
+# Copy and modify the constriants if necesssary.
+function compute_inface_extreme_point!(lmo::MathOptLMO{OT}, direction, x;solve_data=Dict(), kwargs...) where {OT}
+    lmo2 = copy(lmo)
+    MOI.set(lmo2.o, MOI.Silent(), true)
+    variables = MOI.get(lmo2.o, MOI.ListOfVariableIndices())
+    MOI.add_variables(lmo2.o, length(variables))
+    terms = [MOI.ScalarAffineTerm(d, v) for (d, v) in zip(direction, variables)]
+    obj = MOI.ScalarAffineFunction(terms, zero(Float64))
+    MOI.set(lmo2.o, MOI.ObjectiveFunction{typeof(obj)}(), obj)
+    for (F, S) in MOI.get(lmo2.o, MOI.ListOfConstraintTypesPresent())
+        valvar(f) = x[f.value]
+        const_list = MOI.get(lmo2.o, MOI.ListOfConstraintIndices{F,S}())
+        for c_idx in const_list
+            func = MOI.get(lmo2.o, MOI.ConstraintFunction(), c_idx)
+            val = MOIU.eval_variables(valvar, func)
+            set = MOI.get(lmo2.o, MOI.ConstraintSet(), c_idx)
+            if S <: MOI.GreaterThan
+                if set.lower ≈ val
+                    MOI.delete(lmo2.o, c_idx)
+                    func_dict = Dict(field => getfield(func, field) for field in fieldnames(typeof(func)))
+                    
+                    # Get the list of constraints with same ConstraintFunction but LessThan ConstraintSet.
+                    const_list_less = MOI.get(lmo2.o, MOI.ListOfConstraintIndices{F,MOI.LessThan{Float64}}())
+                    
+                    # Check if the ConstraintFunction has other ConstraintSet.
+                    # If exists, delete the constraint to avoid conflict.
+                    for c_idx_less in const_list_less
```

Comment:
There should be a way to ask MOI directly if it sees a `LessThan` constraint on the certain variable index.